src/compiler/emitter.ts

```diff
@@ -4077,6 +4071,18 @@ export function createPrinter(printerOptions: PrinterOptions = {}, handlers: Pri
         writeTrailingSemicolon();
     }
 
+    function emitTypeImportAttributes(node: ImportAttributes) {

```

Comment:
I feel like this function name (and new enum value name) may be misleading; they don't have anything to do with type imports. Maybe this should be `emitImportTypeNodeAttributes`?

Reply:
Renamed.